EIPS/eip-1485.md

```diff
@@ -0,0 +1,200 @@
+F(00,0e)::VEC(0, e, ffffffff, 0)::      FNV  :0000000e, DF=00000003(02) DS(00000001), FNV1 :0e00160a, DF=0300027d(09) DS(01000193), FNV1a:037681c3, DF=07000295(08) DS(01000193), FNV1c:1810d981, DF=0d000d49(09) DS(030004b9),         F___RC=10dfc718, DF:263d8699(15) , F1__RC=a8466ac8, DF:4f6b74db(20) , F1A_RC=93e667bf, DF:856b2efb(19) , F1C_RC=76f80ee3, DF:1343c142(11)
+F(00,0f)::VEC(0, f, ffffffff, 0)::      FNV  :0000000f, DF=00000001(01) DS(00000001), FNV1 :0f00179d, DF=01000197(07) DS(01000193), FNV1a:02768030, DF=010001f3(08) DS(01000193), FNV1c:1710d7ee, DF=0f000e6f(13) DS(01000193),         F___RC=eadd4caf, DF:fa028bb7(17) , F1__RC=695fb77d, DF:c119ddb5(17) , F1A_RC=0f485682, DF:9cae313d(17) , F1C_RC=3667e8dc, DF:409fe63f(18)
+F(00,10)::VEC(0, 10, ffffffff, 0)::     FNV  :00000010, DF=0000001f(05) DS(00000001), FNV1 :10001930, DF=1f000ead(13) DS(01000193), FNV1a:01767e9d, DF=0300fead(14) DS(01000193), FNV1c:0210b6df, DF=15006131(09) DS(1500210f),         F___RC=c4dad246, DF:2e079ee9(17) , F1__RC=2a790432, DF:4326b34f(16) , F1A_RC=d10adebd, DF:de42883f(16) , F1C_RC=1ce48e12, DF:2a8366ce(15)
+```
+
+`F(00,01)` : is input x,y 
+
+`VEC(0, 1, ffffffff, 0)`  : is `fnv_reduce` input vector (a,b,c,d)
+
+`FNV  :00000001, DF=00000001(01) DS(00000001)` : 
+  * `FNV(00,01)` result is 00000001 , 
+  * `DF` : is changed bitcounts, compared with previous outputs, in this case prev[00,00] current[00,01] input is 1bit changed, and output result 1bit changed.
+  * `DS` : is distances of previous result and current result , ABS(prev_fnvresult,current_fnvresult).
+
+** Basically, `DF` is higher is best on hash algorithm.
+
+`F___RC=fefffe6d, DF:111e47a9(14)` : `fnv_reduce = fnv(fnv(fnv(a,b),c),d) ` result is fefffe6d , and Different Bits counts are `14` bits.
+
+  
+## Rationale
+
+In case of ethash algorithm, it can't prevent ASIC forever.
+
+And, current ethash algorithm's FNV function is depricated.
+
+So, It needs to be upgraded and it will make current ethash based ASICs obsolete.
+
+And current TETHASHV1 FNV1A implementation is based on most of ethash , which is verified for a long time.
+
+Another propose of big differencing the Ethash algorithm need to crypto analysis for a long times and need to GPU code optimization times.
+
+**Verification and Optimization timeline Examples**
+
+orignal ethminer (2015) -> claymore optimized miner (2016) [1year]
+
+genoil ethminer (2015) -> ethereum-mining/ethminer (2017) [2year]
+
+## Test Results::
+
+Tethash miner has 2~3% of hashrate degrade on GPU, due to more core computation time.
+
+## Copyright
+
+This work is licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-nc-sa/4.0/).

```

Comment:
The template specifies the following. There have been other EIPs that were Accepted and Final without it, but I'm not sure if this is still possible.

## Copyright
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).